### PR TITLE
docs(faq): add FAQ question about RTMP url

### PR DESCRIPTION
**Pull request by gkatsev**
@mister-ben would love your input on this as well.

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -116,6 +117,13 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
 enables the playback of FLV video where the Flash plugin is available. For more information
 on media formats see the [troubleshooting guide][troubleshooting].
 
+## Q: How can I play RTMP video in video.js?
+
+Make sure that the Flash tech is available. Then, just set the rtmp source with
+an appropriate type like `rtmp/mp4`.
+The main thing to be aware of is that video.js splits the connection url and stream name with the `&` character.
+So, you'd want to update the url to follow that format. For example: `http://example.com/live&foo`.
```
Comment (mister-ben) [inline docs/faq.md]: `rtmp://example.com/live&foo` or `rtmp://example.com/fms&mp4:path/to/file.mp4`.

If the server requires query parameters for authentication, these should be added to the connection part url, for example `rtmp://example.com/live?token=1234&foo`.

File: docs/faq.md [hunk 2]
```diff
@@ -116,6 +117,13 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
 enables the playback of FLV video where the Flash plugin is available. For more information
 on media formats see the [troubleshooting guide][troubleshooting].
 
+## Q: How can I play RTMP video in video.js?
+
+Make sure that the Flash tech is available. Then, just set the rtmp source with
```
Comment (mister-ben) [inline docs/faq.md]: "...is available (RTMP is not playable on browsers without Flash including mobile)."

File: docs/faq.md [hunk 3]
```diff
@@ -116,6 +117,13 @@ techs made available to video.js. For example, video.js 5 includes the Flash tec
...
+an appropriate type like `rtmp/mp4`.
```
Comment (mister-ben) [inline docs/faq.md]: "the appropriate type: `rtmp/mp4` or `rtmp/flv`."

It's a short enough list to just list them.

**Comment by gkatsev** [comment]
Thanks @mister-ben, I've made updates.

**Pull request merged**